homeassistant/components/trafikverket_weatherstation/config_flow.py

```diff
@@ -0,0 +1,79 @@
+"""Adds config flow for Trafikverket Weather integration."""
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_API_KEY, CONF_MONITORED_CONDITIONS, CONF_NAME
+import homeassistant.helpers.config_validation as cv
+
+SENSOR_TYPES: dict[str, str] = {
+    "air_temp": "Air temperature",
+    "road_temp": "Road temperature",
+    "precipitation": "Precipitation type",
+    "wind_direction": "Wind direction",
+    "wind_direction_text": "Wind direction text",
+    "wind_speed": "Wind speed",
+    "wind_speed_max": "Wind speed max",
+    "humidity": "Humidity",
+    "precipitation_amount": "Precipitation amount",
+    "precipitation_amountname": "Precipitation name",
+}
+DOMAIN = "trafikverket_weatherstation"
```

Comment:
Move to ./const.py and import from there, here and in __init__.py to avoid the duplication